2021-05-31 21:46:00,026 INFO    MainThread:19120 [wandb_setup.py:_flush():69] setting env: {}
2021-05-31 21:46:00,026 INFO    MainThread:19120 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-31 21:46:00,026 INFO    MainThread:19120 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\Henrik_Brådland\Google Drive\Studier\Semester 10\IKT-442\wandb\run-20210531_214600-2qbeun3f\logs\debug.log
2021-05-31 21:46:00,027 INFO    MainThread:19120 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\Henrik_Brådland\Google Drive\Studier\Semester 10\IKT-442\wandb\run-20210531_214600-2qbeun3f\logs\debug-internal.log
2021-05-31 21:46:00,027 INFO    MainThread:19120 [wandb_init.py:init():369] calling init triggers
2021-05-31 21:46:00,027 INFO    MainThread:19120 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-05-31 21:46:00,028 INFO    MainThread:19120 [wandb_init.py:init():418] starting backend
2021-05-31 21:46:00,028 INFO    MainThread:19120 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn, using: spawn
2021-05-31 21:46:00,035 INFO    MainThread:19120 [backend.py:ensure_launched():123] starting backend process...
2021-05-31 21:46:00,096 INFO    MainThread:19120 [backend.py:ensure_launched():127] started backend process with pid: 19268
2021-05-31 21:46:00,097 INFO    MainThread:19120 [wandb_init.py:init():423] backend started and connected
2021-05-31 21:46:00,098 INFO    MainThread:19120 [wandb_init.py:init():465] updated telemetry
2021-05-31 21:46:00,099 INFO    MainThread:19120 [wandb_init.py:init():484] communicating current version
2021-05-31 21:46:00,899 INFO    MainThread:19120 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-31 21:46:00,899 INFO    MainThread:19120 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-31 21:46:01,127 INFO    MainThread:19120 [wandb_init.py:init():522] starting run threads in backend
2021-05-31 21:46:02,692 INFO    MainThread:19120 [wandb_run.py:_console_start():1524] atexit reg
2021-05-31 21:46:02,692 INFO    MainThread:19120 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.WRAP
2021-05-31 21:46:02,693 INFO    MainThread:19120 [wandb_run.py:_redirect():1414] Wrapping output streams.
2021-05-31 21:46:02,693 INFO    MainThread:19120 [wandb_run.py:_redirect():1438] Redirects installed.
2021-05-31 21:46:02,693 INFO    MainThread:19120 [wandb_init.py:init():546] run started, returning control to user process
2021-05-31 21:46:02,928 INFO    MainThread:19120 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-05-31 21:46:02,928 INFO    MainThread:19120 [wandb_run.py:_restore():1466] restore
2021-05-31 21:46:03,147 INFO    MainThread:19120 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 581
  total_bytes: 581
}

2021-05-31 21:46:04,167 INFO    MainThread:19120 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 581
  total_bytes: 2927
}

2021-05-31 21:46:04,559 INFO    MainThread:19120 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2927
  total_bytes: 2927
}

2021-05-31 21:46:04,673 INFO    MainThread:19120 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2927
  total_bytes: 2927
}

2021-05-31 21:46:04,782 INFO    MainThread:19120 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2927
  total_bytes: 2927
}

2021-05-31 21:46:04,892 INFO    MainThread:19120 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2927
  total_bytes: 2927
}

2021-05-31 21:46:05,016 INFO    MainThread:19120 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2927
  total_bytes: 2927
}

2021-05-31 21:46:05,127 INFO    MainThread:19120 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2927
  total_bytes: 2927
}

2021-05-31 21:46:06,416 INFO    MainThread:19120 [wandb_run.py:_show_files():1838] logging synced files
